.travis.yml

```diff
@@ -1,5 +1,10 @@
 ---
 language: ruby
-rvm: 2.1.2
-before_script: bundle exec appraisal install
-script: bundle exec appraisal rspec
+rvm:
+  - 2.1.8
```

Comment:
We had to pull this back to 2.1.8 since 2.1.9/2.1.10 are not working on Travis right now.